sportsreference/nfl/schedule.py

```diff
@@ -282,11 +283,19 @@ def datetime(self):
     def result(self):
         """
         Returns a ``string`` constant indicating whether the team won or lost
-        the game.
+        the game.  
+        NFL games may end in a tie if the score is even at the end of OT.  
```

Comment:
This is a bit of a nit-pick, but would it be possible to wrap this text with the previous line? So, something like:
```python
"""
Returns a ``string`` constant …
the game. NFL games may end in a …
"""
```
I've had issues in the past with the auto-documentation with a singular newline like this, and found it's easiest to just wrap together for another sentence. Let me know if you have any issues with this though! Also, looks like this sentence could use a period after the end after None.

Reply:
Ah, didn't know about the auto-doc issue. I will update this